fiftyone/operators/delegated.py

```
@@ -72,35 +76,42 @@ def _execute_operator_in_child_process(
+                    operation_id,
+                )
+                return
+            if log:
+                logger.info(
+                    "\nRunning operation %s (%s) in child process",
+                    operation.id,
+                    operation.operator,
+                )
 
-        result = asyncio.run(service._execute_operator(operation))
+            result = asyncio.run(service._execute_operator(operation))
```

Pylint (4.0.2)

[warning] Access to a protected member _execute_operator of a client class

(W0212)